gui/src/2D/controls/container.ts

```diff
@@ -371,11 +371,12 @@ export class Container extends Control {
 
                         if (this.adaptWidthToChildren && child._width.isPixel) {
                             computedWidth = Math.max(computedWidth, child._currentMeasure.width + child.paddingLeftInPixels + child.paddingRightInPixels);
+                            this.parent?._markAsDirty();

```

Comment:
can we store the value of computedWitdh before the previous line and only call markasdirty if the value changed?

Reply:
sure can.